### ISSUE TITLE
Evaluate retrievial model with MRR

### ISSUE THREAD

**Issue by loretoparisi** [tensorflow/recommenders#118]
Is it possibile to add an example how to evaluate the trained retrieval model with MRR@K in addition to [FactorizedTopK](https://www.tensorflow.org/recommenders/api_docs/python/tfrs/metrics/FactorizedTopK)?
I have found that [tensorflow ranking](https://github.com/tensorflow/ranking) has MRR evaluation.

Thank you.

**Comment by maciejkula** [comment]
Yup, you can definitely use the MRR metric from TensorFlow Ranking. For a retrieval model, you should be able to add it to the list of metrics used by the `FactorizedTopK` layer in TFRS:

```python
import tensorflow_ranking as tfr

task = tfrs.tasks.Retrieval(metrics=tfrs.metrics.FactorizedTopK(
    movies.batch(128).map(movie_model),
    k=100,
    metrics=[
        tfr.keras.metrics.MRRMetric(),
        tf.keras.metrics.TopKCategoricalAccuracy(k=100)
    ],
  )
)
```

```

**Comment by loretoparisi** [comment]
@maciejkula that's amazing, thank you. Closing then.

**Comment by deychak** [comment]
This is no longer possible? Is there a better way to accomplish the same?

**Comment by schistyakov** [comment]
How to implement the same on modern version of TF recommenders? There is no metrics param within tfrs.metrics.FactorizedTopK anymore
